staff/pedro-garriganogues/gamma-template/sass.sass

```diff
@@ -0,0 +1,235 @@
+.burger
+  border-radius: 7px
```

Comment:
You could try to reorder you classes to have a more tidy and comprehensive code.